pkg/util/node.go

```diff
@@ -28,7 +28,7 @@ func GetHostname(hostnameOverride string) string {
 	if string(hostname) == "" {
 		// Note: We use exec here instead of os.Hostname() because we
 		// want the FQDN, and this is the easiest way to get it.
```

Comment:
Should probably update this comment.

Is there a reason we can't use os.HostName() now? I'd be fine changing the comment to a "TODO: figure out if we can just use os.Hostname()"